frontend/src/app/home/AudioInput.tsx

```
@@ -0,0 +1,228 @@
+import { useAppContext } from '@contexts/AppContext';
+import { ReviewType } from '@constants/common';
+import { MediaRecorder } from '@constants/media';
+import { useReactMediaRecorder } from 'react-media-recorder';
+import { SvgIcon } from '@components/ui/svg-icon';
+import Mic from '@assets/icons/mic.svg';
+import Stop from '@assets/icons/stop.svg';
+import Play from '@assets/icons/play.svg';
+import Pause from '@assets/icons/pause.svg';
+
+// Define valid review types as literal types
+type ReviewType = 'perfReview' | 'selfReview';
```

Biome

[error] Shouldn't redeclare 'ReviewType'. Consider to delete it or rename it.

'ReviewType' is defined here:


(lint/suspicious/noRedeclare)